src/oc_erchef/apps/chef_objects/src/chef_regex.erl

```diff
@@ -58,7 +58,8 @@
 %% http://wiki.chef.io/display/chef/Roles#Roles-name.  Judging
 %% from the cookbook names and recipes in the opscode/cookbooks
 %% repository, this regular expression applies to them as well.
--define(NAME_REGEX, "[.[:alnum:]_-]+").
+-define(NAME_CHAR_CLASS, "[.[:alnum:]_-]").
+-define(NAME_REGEX, ?NAME_CHAR_CLASS ++ "+").
```

Comment:
It's minor, but I don't know if the precompile will resolve this - the append might be done inline on each call. Same for the others. May not be significant enough to matter.